### ISSUE TITLE
GeoLite2 database is no longer available publicly

### ISSUE THREAD

**Issue by leplatrem** [mozilla/normandy#2069]
An account is required on the maxmind.com website, and an API required for download.

Shall we do that? Shall we remove the GeoIP parts of Normandy instead?

**Comment by mythmon** [comment]
@sciurus do you have any opinions here? I expect we already have an account that we could use to download these artifacts? It would be nice to have some sort of DB, even if it doesn't have accurate data, for our tests.

**Comment by leplatrem** [comment]
If you already have an account and a different flow to download and update the DB in prod, then maybe we can push an old version in the repo for the dev/tests (they were released under Creative Commons before 30th dec 2019).

**Comment by sciurus** [comment]
For Normandy specifically, I think it is safe to remove the geoip functionality at this point if you want to. I don't see why we would move away from https://github.com/mozilla/classify-client.

For Classify Client, as mentioned in Slack I think setting up a new account to download the noncommercial DB during CI, or checking in a version of the db from before this license change, are both reasonable.

**Comment by mythmon** [comment]
I'm working on figuring out if Maxmind's license allows us to do something like checking in a version of the db. In the meantime, I think removing the geolocation functionality from this repo is a decent idea. To get unblocked I'm going to disable this in CI, and we can do the above as a longer term thing.
